_includes/headertop.html

```diff
@@ -12,6 +12,16 @@
     <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
+    <meta property="og:site_name" content="{{ site.title }}"/>
+    {% if page.title %}<meta property="og:title" content="{{ page.title }}"/>{% endif %}
```

Comment:
The `og:title` property is **required** by the protocol, but we provide it only if the page has a title. This raises two questions:
- Do we have pages without title?
- What about using `site.title` as a fallback in such a case?

Reply:
`page.title` is Jekyll's.
Not sure it is required from https://jekyllrb.com/docs/variables/
Added fallback in https://github.com/scala/docs.scala-lang/pull/1468/commits/62ef7f0c76fad3917a0645585c87a776edf4a1d7